clients/js-legacy/test/exports/module.mjs

```diff
@@ -0,0 +1,10 @@
+// This ensures that we do not rely on `__dirname` in ES modules even when it is polyfilled.
+globalThis.__dirname = 'DO_NOT_USE';
+
+import { Keypair } from '@solana/web3.js';
+import { createInitializeInstruction } from '../../dist/src/index.js';
```

Comment:
Your imports don't seem to match with the paths set in the `package.json`. For instance, I think this should be `../../lib/esm/index.js`?

Reply:
Ah yes, this was what I was confused about since you mentioned `dist` in a previous comment. I fixed it, and ow I see that the cjs export is failing 😅  . I'll try to see why this is. It would also be interesting to check if the token-2022 js-legacy fails this test since it has the exact same tsconfig.

Reply:
I wouldn't be surprised if it does tbh if it uses `tsc` for exporting dual packages (ESM+CJS).